MatCalcitePlugin/src/com/github/vlsi/mat/calcite/cli/MCPApplication.java

```diff
@@ -0,0 +1,48 @@
+package com.github.vlsi.mat.calcite.cli;
+
+import sqlline.Application;
+import sqlline.CommandHandler;
+import sqlline.MCPSqlLine;
+import sqlline.SqlLine;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+public class MCPApplication extends Application {
+  private static final Set<String> ALLOWED_SQLLINE_COMMANDS = new HashSet<String>() {{
```

Comment:
@snuyanzin , I might be late here :) Do you remember by chance why do you limit the set of commands?

Reply:
As far as I remember there are database specific commands e.g. `!isolation`, '!drop` and etc. 
The idea was to limit to set of commands which makes sense for mat-calcite-plugin.

Agree that the list of commands itself is the subject for discussion.